client/components/formatted-header/index.jsx

```diff
@@ -29,7 +29,7 @@ function FormattedHeader( { headerText, subHeaderText } ) {
 }
 
 FormattedHeader.propTypes = {
-	headerText: PropTypes.string,
+	headerText: PropTypes.oneOfType( [ PropTypes.string, PropTypes.array ] ),
```

Comment:
[`prop-types` have a `node` type](https://github.com/facebook/prop-types), too - which might be a better fit here:
```
  // Anything that can be rendered: numbers, strings, elements or an array
  // (or fragment) containing these types.
```

Reply:
Smart. :)

Reply:
Actually, after looking at it, string/array makes more sense. Narrows it down, and it's actual text, just that `translate` returns an array.